Lib/test/typed_dict.py

```diff
@@ -0,0 +1,19 @@
+"""Script used to test `get_type_hints()` on a class formed by inheriting
+a `TypedDict` with postponed annotations, for bpo-41249.
```

Comment:
Can you rename the file to something like typed_dict_helper.py? I want it to be very clear that this is not seen as something exported by the test package. (Maybe even _typed_dict_helper.py?)

I also don't think that the bpo number needs to be mentioned in the docstring -- one can get that from the commit message. (It'd be lovely if the summary fit on one line, but I'm not holding you to that. :-)